src/exports.jl

```diff
@@ -111,6 +111,7 @@ export
     ndofs_per_cell,
     celldofs!,
     celldofs,
+    celldofsview,
```

Comment:
Don't think this needs to be exported, and I don't think you need it internally either, use CellCache (possibly with more fine grained update flags to only update the dofs).